Sources/SwiftInspectorTestHelpers/Tests/VisitorExecutorSpec.swift

```diff
@@ -47,46 +47,20 @@ final class VisitorExecutorSpec: QuickSpec {
   }
 
   override func spec() {
-    describe("createFile(withContent:andWalk:)") {
+    describe("walkVisitor(_:overContent:)") {
```

Comment:
```suggestion
    describe("walk(:)") {
```

since we're actually not testing the `walkVisitor` method, but the underlying walk method.